README.md

```diff
@@ -7,8 +7,7 @@ Before getting started, make sure you have the following tools installed:
 
 - [nvm (Node Version Manager)](https://github.com/nvm-sh/nvm): To manage Node.js versions.
 - [Node.js](https://nodejs.org/): Javascript runtime environment.
-- [npm](https://www.npmjs.com/): Package manager for Node.js.
-- [sdkman](https://sdkman.io/): Software Development Kit Manager.
```

Comment:
Why are you removing this?

Reply:
@stephane-segning  Surely a mistake from @Tekum-Emmanuella during the documentation. We began by including it in the README file then later on decided to create our own doc

Reply:
Ok.

Reply:
It should be fixed then